wcfsetup/install/files/acp/templates/packageUpdate.tpl

```diff
@@ -1,56 +1,51 @@
-{include file='header' pageTitle='wcf.acp.package.update.title'}
-
-<script data-relocate="true">
-	$(function() {
-		WCF.Language.addObject({
-			'wcf.acp.package.update.excludedPackages': '{jslang}wcf.acp.package.update.excludedPackages{/jslang}',
-			'wcf.acp.package.update.title': '{jslang}wcf.acp.package.update.title{/jslang}',
-			'wcf.acp.package.update.unauthorized': '{jslang}wcf.acp.package.update.unauthorized{/jslang}'
-		})
-		
-		new WCF.ACP.Package.Update.Manager();
-	});
-</script>
+{include file='header' pageTitle='wcf.acp.package.update.availableUpdates'}
 
 <header class="contentHeader">
-	<h1 class="contentTitle">{lang}wcf.acp.package.update.title{/lang}</h1>
+	<h1 class="contentTitle">{lang}wcf.acp.package.update.availableUpdates{/lang} <span class="badge badgeInverse">{#$items}</span></h1>
 </header>
 
-{foreach from=$availableUpdates item=update}
-	<section class="section jsPackageUpdate" data-package="{$update[package]}">
-		<header class="sectionHeader">
-			<h2 class="sectionTitle"><label>
-				<input type="checkbox" value="1" checked>
-				{$update[packageName]|language}
-			</label></h2>
-			{if $update[packageDescription]}<p class="sectionDescription">{$update[packageDescription]|language}</p>{/if}
-		</header>
-		
-		<dl>
-			<dt>
-				{lang}wcf.acp.package.installedVersion{/lang}
-			</dt>
-			<dd>
-				{$update[packageVersion]} ({$update[packageDate]|date})
-			</dd>
-		</dl>
-		<dl>
-			<dt>
-				{lang}wcf.acp.package.availableVersions{/lang}
-			</dt>
-			<dd>
-				<select>
-					{foreach from=$update[versions] item=version}
-						<option value="{@$version[packageVersion]}"{if $version[packageVersion] == $update[version][packageVersion]} selected{/if}>{$version[packageVersion]} ({@$version[packageDate]|date})</option>
-					{/foreach}
-				</select>
-			</dd>
-		</dl>
-	</section>
-{/foreach}
+<div class="section">
+	<div class="contentItemList packageUpdateList">
+		{foreach from=$availableUpdates item=update}
+		<div class="contentItem packageUpdate" data-package="{$update[package]}" data-version="{$update[newVersion][packageVersion]}">
+			<div class="contentItemContent">
+				<div class="contentItemTitle">{$update[packageName]|language}</div>
+				<div class="contentItemDescription">
+					<small{if $update[newVersion][servers][0][packageUpdateServerID] === $woltlabUpdateServer->packageUpdateServerID} class="packageSearchAuthorWoltlab"{/if}>{$update[author]}</small>
+					<p class="packageUpdateAvailable">
+						{$update[packageVersion]}
+						→
```

Comment:
It would be desirable if the full `{$from} → {$to}` construct would be part of a language item to allow proper localization into languages where such an arrow notation does not exist.